.github/workflows/ci.yml

```
@@ -22,7 +22,7 @@ jobs:
     - uses: actions/setup-go@v5
       with:
         go-version: ${{ matrix.go-version }}
-    - uses: goreleaser/goreleaser-action@v2
+    - uses: goreleaser/goreleaser-action@v6
       with:
         args: release
```

```suggestion
        args: release --snapshot
```

And add `version: 2` to the top of the file `.goreleaser.yml`.